ipatests/test_integration/test_trust.py

```diff
@@ -90,6 +92,49 @@ def remove_trust(self, ad):
         tasks.remove_trust_with_ad(self.master, ad.domain.name)
         tasks.clear_sssd_cache(self.master)
 
+    def check_filter_users(self, realm, aduser, provider=None, offline=False):
+        master = self.master
+        user1 = 'some_user@some.domain'
+        tasks.backup_file(self.master, paths.SSSD_CONF)
+        filter_users = {'filter_users': '{0},{1}'.format(user1, aduser)}
+        log_file = '{0}/sssd_nss.log'.format(paths.VAR_LOG_SSSD_DIR)
+        if provider:
```

Comment:
modify_sssd_conf has default value 'ipa'
So instead of writing these 4 lines  you can simply do:
```
def check_filter_users(...., provider='ipa',....):
    ....
    tasks.modify_sssd_conf(master, master.domain.name, filter_users, provider)
    ....
```